dataset/datasets.py

```diff
@@ -350,7 +444,6 @@ def hermit_polynomial(treatment, gams):
         # h(a) = gam[0] + gam[1] * a + gam[2] * (a**2 - 1) + gam[3] * (a***3 - 3 * a)
         hermit = hermit_polynomial(torch.logit(t), gams) + x @ beta
 
-
         y = hermit + error
 
         #! Dimeji:  I am assuming we don't need any noise in this case, so I set noise to None
```

Comment:
We do need noise!
 `noise=erro`r in `y = hermit + error`
 Here noise means the variation from one sample to the other. For the same individual, the term `hermit` would change with another treatment, but `error` is constant. Hence when computing counterfactuals, we need to pass `error` as the `noise` argument.
`